react/features/base/label/components/AbstractCircularLabel.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import { Component } from 'react';
+
+export type Props = {
+
+    /**
+     * String that will be rendered as the label itself.
+     */
+    label: string
+};
+
+/**
+ * Abstract class for the {@code CircularLabel} component.
+ */
+export default class AbstractCircularLabel<P: Props> extends Component<P> {
```

Comment:
Do you think it's still good to have this abstract if there isn't any abstract/base methods? I suppose the shared props are good but I wonder if it's enough.

Reply:
Very good point, actually I was thinking about the same. Then I realised that when working with a real object oriented language, majority of the implemented inheritance is about property inheritance and there is a much fewer cases when you inherit behaviour. I like inheritance though, that's why I went with inheritance.

Taking the point further however (and this is now not about this particular change or comment), Facebook discourages anyone using inheritance in react at all. Which would be a good reason not to do it, we however already built our app over inheritance and code sharing over abstract components, so it would be weird to stop using it now. Maybe would be good to have a longer discussion about it though, but I think I would still be on the inheritance side :)